torch/_meta_registrations.py

```diff
@@ -1166,6 +1167,10 @@ def meta_binop_inplace_alpha(self, other, alpha=1):
     return self
 
 
+@register_meta([aten.round.default, aten.round.decimals])
+def meta_round(self, **kwargs):
+    return _elementwise_meta(input)
```

Comment:
@mruberry @ezyang is it ok to reuse this function that's used to find the correct meta information (strides, dtype). It seems like the meta and prim implementation for all elementwise functions should be the same?